From fc597c8c8b2e74bc7609a778adee924c69fca4f4 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Thu, 30 Mar 2017 18:11:50 +0900
Subject: [PATCH 2417/2848] HADOOP-14256. [S3A DOC] Correct the format for
 "Seoul" example. Contributed by Brahma Reddy
 Battula.

(cherry picked from commit 0d053eeb3002f348d34cf3aac5b43333af8ff10a)
(cherry picked from commit bb187e145a851e734b00a31d9df2efe31a8ac214)

Change-Id: I0c39f930078ecdea6ea10c7eacc3aab07d701465
---
 .../src/site/markdown/tools/hadoop-aws/index.md    |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md b/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md
index e6e31e2..cf8fa2f 100644
--- a/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md
+++ b/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md
@@ -1053,6 +1053,7 @@ Frankfurt
 ```
 
 Seoul
+
 ```xml
 <property>
   <name>fs.s3a.endpoint</name>
-- 
1.7.9.5

